backends/optimized/ceed-opt-basis.c

```diff
@@ -82,6 +84,7 @@ static int CeedBasisApply_Opt(CeedBasis basis, CeedInt nelem,
       post *= Q;
     }
   } break;
+  // Evaluate the gradient to/from quadrature points
```

Comment:
It's worth asking whether comments like this are providing new information or just rephrasing the code. Comments get stale easily so it's better if the code is clear with fewer comments.

Reply:
Good point. Maybe leaving it in the Ref backend as that's the 'educational' backend and clearing it out of the Opt backend?

Reply:
Sounds reasonable.